node/src/reactor/validator.rs

```diff
@@ -185,6 +186,9 @@ pub enum Event {
     /// Chainspec loader announcement.
     #[from]
     ChainspecLoaderAnnouncement(#[serde(skip_serializing)] ChainspecLoaderAnnouncement),
+    /// Blocklist announcement.
+    #[from]
+    BlocklistAnnouncement(BlocklistAnnouncement<NodeId>),
```

Comment:
Most variants `#[serde(skip_serializing)]` — not sure which ones should and shouldn't?

Reply:
`skip_serializing` is used for values where there is no `Serialize` implementation. Some might not have been updated.